### DISCUSSION TITLE
I add support for block comments in Markdown configuration

### DISCUSSION THREAD

**Discussion by Rusydy** [zed-industries/zed#22353] [Show and tell]
I would like to have a shortcut for comment in markdown file, then I create this PR https://github.com/zed-industries/zed/pull/22352.

**Comment by Rusydy** [comment]
Hi @SomeoneToIgnore 

May I know what do you think about this?

**Reply by SomeoneToIgnore** [reply]
Have not thought about it at all.

Is it true, that those comments are not standard and are part of certain markdown extensions as [gfm](https://github.github.com/gfm/)?

The PR seems to have issues with CI, presumably a rebase might help.

**Reply by Rusydy** [reply]
I usually use these comments on other editor and GitHub, [GitHub docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments)

for the PR, I already rebase it.

**Reply by Rusydy** [reply]
Thank you, @SomeoneToIgnore 

After rebase, the CI is all done with green status

**Reply by SomeoneToIgnore** [reply]
Ok then, let's give it a try.
I'm still unsure whether the comment syntax is widespread enough, but let's see.
